doc/howto/network_acls.md

```diff
@@ -836,7 +836,7 @@ lxc config device set ubuntu-container ovn-nic security.acls="web-traffic,intern
 
 #### View the existing NIC configuration
 
-To update the configuration for an instance's device using the API, you must include all the required keys for the device—even if you're only changing one key. The required keys are the `type` of `nic` and the `network` name. Include these along with the key to update, and any other existing keys (unless you want to remove them). Omitted keys are reset to default values.
+To update the configuration for an instance's device using the API, you must include all the required keys for the device—even if you're only changing one key. For an instance NIC device, the required keys are the `type` of `nic` and the `network` name. Include these along with the key to update, and any other existing keys (unless you want to remove them). Omitted keys are reset to default values.
```

Comment:
Is this section modified twice within the PR? We should avoid this an ensure each commit in the PR is the final form for review.